canonical-web-and-design/ubuntu.com
# Copy update on /support

## Done

- Rebuild `/support` page

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/support
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check page against [copy doc](https://docs.google.com/document/d/1AU2dwTXpQk2UuEvSuAqRKrbPWYb-LR0ZGvWu6xw95I8/edit#)


## Issue / Card

Fixes [#3365](https://github.com/canonical-web-and-design/web-squad/issues/3365)
